pubs

```diff
@@ -1,3 +1,5 @@
-ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBIml4uRn9opZLX9/UisOGCRvT/Q8uxUUw9RlHbzjINNaXeswmQqq0DmX6zT9n1CfajWf6FY9j3GB4q/El2lP7KU= thoth@flux.thalhalla.net
-
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDAgzNBMM7hJKeJPhcP2Vp2gEUjzU8L3lewyycOHaaxyAcT62J3J1/UGTKpFDMsvGi/+62KQz9DH4BgzC4vnRn/Xrss234vKLo8QvdpycUexN0m1GcyUjrl3IueZzGAZpMMLLb92yRqSyxHEMPvuddRcVtBkOL7aLEx+mk3c2NqldONlDvS0KjI6kswhKDhFE+0YQhqz/lLTOWH+AryIVGe9UYLRxMen7guhTXgYtisXIqI+AZbHfL3CEvYO9nQpXla0EUxdi20R3iS5yZy1cUtJ64rK1i10K01a2dqlyTId5jygwzL2rWmcdrPNDXTXGCcSGg77gUMYaiDJLvyb+pfPumlaY8v2hUNFfFoiRCqqhbwRGvgWB4F438W4eT6ZhLnQ3qVcQ8I7HKM+C7YH3x4uTg9X0H5ZQZgrkSQxcFfFv1l+rKvdLUQ8iWVFyxU/1uCFW83nHcoYLa6pjaoxzmzPeFsm7PxoFiIQQz0IVjDcyfsVte4T8GqkT5nJ0c7QdOxzQTK1GCjoIfc2onSS+8HKnExxaJ4tpLcu+ez4h7wtjhofe3Gjemi6JQ//OTnZ15rOkugzjASgjVoe2tGKRTZMO2/WDRKzGJIZ9OwK1ycn/DIY/cSyJk7+1UqcD2dgZsCDf6m1f6YbzCdduKOyeJqT/pAkIBvEU1HRwO3I52N2w== orie@transmute.industries
\ No newline at end of file
+# Or13
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDAgzNBMM7hJKeJPhcP2Vp2gEUjzU8L3lewyycOHaaxyAcT62J3J1/UGTKpFDMsvGi/+62KQz9DH4BgzC4vnRn/Xrss234vKLo8QvdpycUexN0m1GcyUjrl3IueZzGAZpMMLLb92yRqSyxHEMPvuddRcVtBkOL7aLEx+mk3c2NqldONlDvS0KjI6kswhKDhFE+0YQhqz/lLTOWH+AryIVGe9UYLRxMen7guhTXgYtisXIqI+AZbHfL3CEvYO9nQpXla0EUxdi20R3iS5yZy1cUtJ64rK1i10K01a2dqlyTId5jygwzL2rWmcdrPNDXTXGCcSGg77gUMYaiDJLvyb+pfPumlaY8v2hUNFfFoiRCqqhbwRGvgWB4F438W4eT6ZhLnQ3qVcQ8I7HKM+C7YH3x4uTg9X0H5ZQZgrkSQxcFfFv1l+rKvdLUQ8iWVFyxU/1uCFW83nHcoYLa6pjaoxzmzPeFsm7PxoFiIQQz0IVjDcyfsVte4T8GqkT5nJ0c7QdOxzQTK1GCjoIfc2onSS+8HKnExxaJ4tpLcu+ez4h7wtjhofe3Gjemi6JQ//OTnZ15rOkugzjASgjVoe2tGKRTZMO2/WDRKzGJIZ9OwK1ycn/DIY/cSyJk7+1UqcD2dgZsCDf6m1f6YbzCdduKOyeJqT/pAkIBvEU1HRwO3I52N2w== orie@transmute.industries
+# Josh
+ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBBI0wpMByzpFXTHLV98y4lFm8qB/XrLP8417Db0P0OuCgpQ/+RmaATZ9NjlDgs5jGX8UWe9zEJFq77zmlQHqIrA= josh@flux.transmute.network
+ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNv6ctZFflkpPgbjkvYne46Yyd/jgjm/aEPkqc5ju01dooa7G2DzPpp9rw5he3mjqgEULedxJW2az8/try0ealU= josh@stealth.gs70.msi.oltorf.transmute.network
```

Comment:
Can you re-add my pubkey?
https://github.com/transmute-industries/pubs/pull/9/commits/7e6d35a09a16589147a877f8436adddf8bbcf60a

Reply:
This recent merge should've fixed that, I should have pulled apparently last night before squashing and merging.